samples/openthread/coprocessor/prj.conf

```diff
@@ -12,7 +12,7 @@ CONFIG_OPENTHREAD_COPROCESSOR_RCP=y
 CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=2048
 
 # Enable OpenThread features set
-CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER=y
+#CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER=y
```

Comment:
Should this be commented out? or removed?

Reply:
fixed.